### DISCUSSION TITLE
Looking for maintainers

### DISCUSSION THREAD

**Discussion by hawkeye64** [quasarframework/quasar#9560] [Quasar App Extensions]
If maintainers are not found, the following App Extension will become deprecated in the future:

| App Extension | Status |
| ------------------- | --------- |
| QPdfviewer     |   Done    |
| QWindow        |             |
| QActivity         |  Done   |
| QScroller        |              |
| QOverlay        | Done    |
| QFlashcard     | Done   |
| QRibbon         |             |
| QZoom           |             |

Or, you can vote on ones you think should stick around. Comment below and make a case.

**Comment by DaDuFan** [comment]
First, thank you very much for making these App Extensions!

I haven't looked at all Extensions, but I would love to vote for **QWindow**.

Unfortunately I neither have the time to maintain it, nor the knowledge, as I haven't really used Quasar, yet.
But I hope I'll soon start to use it :-) 

All I could offer atm is a small donation.

QOverlay would be nice in combination with QWindow, too.

**Comment by jonathanpmartins** [comment]
I do not use any of the above App Extensions. I was just curious about using ``QZoom``.

**Comment by ndianabasi** [comment]
Hello @hawkeye64. I remember your help at Discord. Thank you.

What does maintaining the extensions involve? Please put me through. I can dedicate some hours per week to maintain few of them if I understand the mechanisms of the extensions.

**Reply by hawkeye64** [reply]
Maintaining means to fix any found bugs. The immediate maintenance needed would be to make any of them Vue 3/Quasar 2 compatible.

**Reply by ndianabasi** [reply]
Okay. I will put one of them on my to-do list as a test-run.

**Reply by hawkeye64** [reply]
@ndianabasi The easiest one would be QOverlay

**Reply by ndianabasi** [reply]
I see that the demo within each extension is based on Quasar v1. How do I test the extension if there is no compatible demo for v2?

**Reply by hawkeye64** [reply]
A new one needs to be created

**Comment by Dirk-** [comment]
My vote goes to QZoom. I use it in my app and like it very much.

Unfortunately, I am just a Quasar/Vue user and would be absolutely lost without these frameworks, so I do not feel comfortable to maintain such an extension.

**Reply by hawkeye64** [reply]
@Dirk- No problem. Thanks for the feedback. I am a bit worried that QZoom may not make the cut. The changes to Vue 3 might make it impossible as QZoom relies on cloning the vNode.

**Comment by nucle** [comment]
Hi,
I would like to help.

**Reply by hawkeye64** [reply]
Let me know which one you have an interest in. I can always mentor if needed

**Reply by nucle** [reply]
Maybe I would start with a simple one. If everything is as u expect then I can start with another one.
Maybe an introduction would be also good.

**Reply by hawkeye64** [reply]
@nucle Ping we next week. I'm on vacation until Sunday...

**Reply by nucle** [reply]
i wrote an email. Let me know when u are back.

**Reply by hawkeye64** [reply]
I was back on Monday, but I will be gone all next week too. I'll look for the email as I am still catching up.

**Reply by hawkeye64** [reply]
@nucle I replied...

**Comment by geoervin** [comment]
A vote for QPdfviewer. It solves many issues with cordova on mobile platforms.

**Reply by CharlieDigital** [reply]
I am interested in `QPdfViewer`

No problem working on upgrading it to Vue3 compat, but may need some coaching and initial setup @hawkeye64 

I have worked with raw pdfjs before, but not with Quasar extensions so just need to be pointed to the right direction.

**Reply by CharlieDigital** [reply]
@geoervin @hawkeye64 

I tested a migration locally.  

Changes are noted here: https://github.com/quasarframework/app-extension-qpdfviewer/issues/87#issuecomment-885984931

Not sure how to go about moving forward since it seems like it should have two versions published.

**Comment by chrisvv24** [comment]
Adding my vote for QPdfViewer. Our organisation uses quasar and pdf functionality quite heavily and I have found this package to be preferable to all the others I scouted.

Please consider keeping it around and maintained for our and Im sure many others sake.
